splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -61,22 +62,16 @@ else if(v1==v2)
         scripts=new TreeMap<>(ddComparator);
         scripts.put(new Splice_DD_Version(sdd,1,0,0),new UpgradeScriptForFuji(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,1,1,1),new LassenUpgradeScript(sdd,tc));
-        scripts.put(new Splice_DD_Version(sdd,2,6,0),new SourceCodeUpgradeScript(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,2,6,0, 9999),new UpgradeScriptFor260(sdd,tc));
```

Comment:
Should be 2.6.0. If the catalog version is less than 2.6.0, then run this upgrade script.